redash/monitor.py

```diff
@@ -35,14 +35,8 @@ def get_queues():
 
 
 def get_queues_status():
-    queues = {}
-
-    for queue in get_queues():
-        queues[queue] = {
-            'size': redis_connection.llen(queue)
-        }
-
-    return queues
+    return dict({queue: {'size': redis_connection.llen(queue)} for queue in get_celery_queues()}.items() +
+                {queue.name: {'size': len(queue)} for queue in Queue.all(connection=redis_connection)}.items())
```

Comment:
```suggestion
                {queue.name: {'size': queue.count} for queue in Queue.all(connection=redis_connection)}.items())
```

Probably better to use `Queue.count` here.

Reply:
Just realized you're using Python's `len` and not `llen`. :man_facepalming: I guess it's my cue to go to sleep. :sleeping: